pxf/pxf-hive/src/main/java/org/apache/hawq/pxf/plugins/hive/HiveORCAccessor.java

```diff
@@ -162,6 +165,10 @@ private boolean buildArgument(SearchArgument.Builder builder, Object filterObj)
         ColumnDescriptor filterColumn = inputData.getColumn(filterColumnIndex);
         String filterColumnName = filterColumn.columnName();
 
+        /* Need to convert java.sql.Date to Hive's DateWritable Format */
+        if (filterValue instanceof Date)
```

Comment:
Hive doesn't support java.sql.Data so I needed to catch this explicitly and convert to DateWritable